inst/include/cpp11/function.hpp

```diff
@@ -68,7 +68,7 @@ class package {
       return R_BaseEnv;
     }
     sexp name_sexp = safe[Rf_install](name);
-    return safe[Rf_findVarInFrame](R_NamespaceRegistry, name_sexp);
+    return safe[detail::r_env_get](R_NamespaceRegistry, name_sexp);
```

Comment:
I am confident about this switch:
- `r_env_get()` uses `Rf_findVarInFrame3(env, sym, TRUE)` on older R
- `Rf_findVarInFrame()` is exactly a wrapper around `Rf_findVarInFrame3(env, sym, TRUE)`

So `r_env_get()` is basically `Rf_findVarInFrame()` with a few extra checks - like erroring if the object is not found, as we want here.

On newer R it is using the "blessed" `R_getVar()` with `inherits = false`, which is also what we want.